docs/MAINTAINERS.md

```diff
@@ -0,0 +1,42 @@
+## For maintainers
+
+### Helper script for creating fortran tests: create\_fortran\_test.py
+
+A easy way to create an exercise test is to use the script
+bin/create\_fortran\_test.py
+
+Use this script to create an initial <exercise>\_test.f90 file which can be used as a skeleton for your test.
+Typically, you will have to replace the 'response'-function in the generated file with the correct function call.
+
+Also note that Fortran has issues with special characters such as `\n` and `\t` so take special care handling these.
+
+#### Prerequsites
+- Working cmake and fortran compiler
```

Comment:
Capitalize these as is standard when referring to these tools, unless you are referring to the executable file.

Reply:
resolved by https://github.com/exercism/fortran/pull/63/commits/ec141866481247cc4205fd3c0c64f0cde3e9a4d8